$NetBSD: patch-aa,v 1.2 2016/03/18 23:13:26 sevan Exp $

Honor PYTHON settings in Documentation Makefile.
Backport of commit e22723c8ee19cf1bb4fdf68b2f1207442ab59e39

Allow ASCIIDOC_EXTRA to be overridden

--- Documentation/Makefile.orig	2009-10-26 16:09:04.000000000 +0000
+++ Documentation/Makefile
@@ -1,4 +1,4 @@
-COMMANDS = $(shell ../stg-build --commands)
+COMMANDS = $(shell $(PYTHON) ../stg-build --commands)
 COMMANDS_TXT = $(patsubst %,stg-%.txt,$(COMMANDS))
 
 MAN1_TXT= stg.txt $(COMMANDS_TXT)
@@ -19,7 +19,7 @@ man1dir=$(mandir)/man1
 # DESTDIR=
 
 ASCIIDOC=asciidoc --unsafe
-ASCIIDOC_EXTRA =
+ASCIIDOC_EXTRA ?=
 INSTALL?=install
 
 #
@@ -63,10 +63,10 @@ clean:
 ALL_PY = $(shell find ../stgit -name '*.py')
 
 $(COMMANDS_TXT): $(ALL_PY)
-	../stg-build --asciidoc $(basename $(subst stg-,,$@)) > $@
+	$(PYTHON) ../stg-build --asciidoc $(basename $(subst stg-,,$@)) > $@
 
 command-list.txt: $(ALL_PY)
-	../stg-build --cmd-list > $@
+	$(PYTHON) ../stg-build --cmd-list > $@
 
 %.html : %.txt
 	$(ASCIIDOC) -b xhtml11 -d manpage -f asciidoc.conf $(ASCIIDOC_EXTRA) $<
